### ISSUE TITLE
Scaffolding model with String @id results in 404 for create page

### ISSUE THREAD

**Issue by RobertBroersma** [redwoodjs/redwood#330]
## Repro:

**Model**
```
model Story {
  id          String @id @default(uuid())
  title       String
}
```

--> Scaffold post.

**Result**
```jsx
// Routes.js

<Route path="/stories" page={StoriesPage} name="stories" />
<Route path="/stories/{id}" page={StoryPage} name="story" />
<Route path="/stories/{id}/edit" page={EditStoryPage} name="editStory" />
<Route path="/stories/new" page={NewStoryPage} name="newStory" />
```

Since `/stories/{id}` receives a string, visiting `/stories/new` tries to find a Story with id `new`.

Moving the `/stories/new` route up above the `/stories/{id}` route fixes it!

**Comment by weaversam8** [comment]
@RobertBroersma - this should have been fixed in #211. Can you confirm you're running on `v0.2.5` and that you followed the [upgrade instructions attached to the release](https://github.com/redwoodjs/redwood/releases/tag/v0.2.5)?

**Comment by RobertBroersma** [comment]
@weaversam8 Ah, I missed that PR!

I am indeed running `v0.2.5`

I installed Redwood and started the tutorial yesterday.

**Comment by weaversam8** [comment]
Huh, interesting. This was what was scaffolded in the Routes file?

**Comment by RobertBroersma** [comment]
Yes it was. I've double checked to see if I'm crazy, opened `commands/scaffold.js` from `node_modules`, and it's indeed correct as in the PR you linked, but for some reason it's generating in reverse order for me.

I've scaffolded several more models and they all end up this way!

**Comment by cannikin** [comment]
`master` has the routes re-ordered and will be released in 0.3.0 soon! `/new` will now be at the top of the list:

```
<Route path="/posts/new" page={NewPostPage} name="newPost" />`
<Route path="/posts/{id}/edit" page={EditPostPage} name="editPost" />`
<Route path="/posts/{id}" page={PostPage} name="post" />`
<Route path="/posts" page={PostsPage} name="posts" />`
```

**Comment by RobertBroersma** [comment]
Cool, thanks for the reply!
